certora/harnesses/SocialRecoveryModuleHarness.sol

```diff
@@ -82,4 +82,13 @@ contract SocialRecoveryModuleHarness is SocialRecoveryModule {
             }
         }
     }
+
+    /**
+     * @notice Returns the hash of all the guardians of a wallet.
+     * @param _wallet The target wallet.
+     * @return guardiansHash The hash of all the guardians of a wallet.
+     */
+    function guardiansHash(address _wallet) public view returns (bytes32) {
+        return keccak256(abi.encodePacked(getGuardians(_wallet)));
```

Comment:
uber-uber-nit:

```suggestion
        return keccak256(abi.encode(getGuardians(_wallet)));
```

Makes absolutely no difference here, but I feel like encode is more correct from a conceptual level. Just ignore if you don't like the suggestion or used packed encoding for a reason.

Reply:
I initially thought to make it `encode(...)`, but here rest assured we can expect guardians to be unique and no exploitation I can think of, thought to keep it `encodePacked(...)`.

Reply:
It was more of a style thing - there are fewer "sharp edges" associated with `encode` over `encodePacked`.

As I mentioned, I also think it makes absolutely no different here.